src/helpers/userHelper.js

```diff
@@ -5,3 +5,7 @@ export const userName = () => {
 export const userExists = () => {
 	return localStorage.getItem('bearerToken') !== null;
 };
+
+export const isAdmin = (user) => {
```

Comment:
It's minor thing but it can be simplified as `
export const isAdmin = (user) => user.role === 'admin';
`